tests/js/setupFramework.ts

```diff
@@ -31,6 +31,11 @@ failOnConsole({
       return true;
     }
 
+    // TODO: Remove this after updating jsdom, currently it cannot handle @container queries
+    if (/Error: Could not parse CSS stylesheet/.test(errorMessage)) {
+      return true;
+    }
```

Comment:
avoiding mocking console.error. but it does appear jsdom throws an error because of container queries.

https://github.com/getsentry/sentry/blob/master/static/app/views/performance/newTraceDetails/trace.spec.tsx#L478-L479